.github/workflows/ci-build.yaml

```diff
@@ -32,7 +32,7 @@ jobs:
       - name: Set up Go
         uses: actions/setup-go@v2
         with:
-          go-version: 1.19
+          go-version: '1.20'
```

Comment:
Do we need to upgrade go version to 1.20 in go.mod to match these updates?

Reply:
its upgraded in a later PR